pkg/codegen/templates/stdhttp/std-http-handler.tmpl

```diff
@@ -3,21 +3,26 @@ func Handler(si ServerInterface) http.Handler {
   return HandlerWithOptions(si, StdHTTPServerOptions{})
 }
 
+type Mux interface {
```

Comment:
You should explain what you wrote in 1718 here. I mean the reason why you define this interface

You should also mention *http.ServeMux implements it

Reply:
> I'm currently implementing open-telemetry, and I want to have the route pattern as part of the operation name.
I would be able to get the method and route pattern by injecting a custom ServeMux during the handler initialization.

Reply:
I know, I meant to write this in the comment of ServeMux interface

But the comment you added is OK, it could have been better, but let's say it's fine for now